NEWS

```diff
@@ -15,6 +15,10 @@ PHP                                                                        NEWS
 - Sockets:
   . Fixed socket constants regression as of PHP 8.2.0beta3. (Bruce Dou)
 
+- Session:
+  . Fixed GH-9584 (PS(mod_user_class_name) must not leak into next request).
```

Comment:
Maybe "Fix leaking of pointer to freed memory when not unregistering custom session handler", or something along those lines.